pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -526,6 +526,48 @@ type PrometheusSpec struct {
 	// Defines the runtime reloadable configuration of the timeseries database
 	// (TSDB).
 	TSDB TSDBSpec `json:"tsdb,omitempty"`
+	// TracingConfig configures tracing in Prometheus. This is an experimental feature, it may change in any upcoming release in a breaking way.
+	// +optional
+	TracingConfig *PrometheusTracingConfig `json:"tracingConfig,omitempty"`
```

Comment:
doesn't it also apply for PrometheusAgent?

Reply:
The trace in prometheus is only instrumented for queries. Since it's not possible to query the agent, I don't think it's possible to configure tracing

Reply:
Oh, apologies!

Let me open an issue to move tracing config to CommonPrometheusFields and make we configure both Server and Agent